setup.py

```diff
@@ -19,16 +19,16 @@
 
 setup(
     name='cloudify-aws-plugin',
-    version='2.3.2',
+    version='2.3.3',
     author='Cloudify Platform Ltd.',
     author_email='hello@cloudify.co',
     license='LICENSE',
     packages=find_packages(exclude=['tests*']),
     description='A Cloudify plugin for AWS',
     install_requires=[
-        'cloudify-common>=4.5',
+        'cloudify-common==5.1.0.dev1',
```

Comment:
Right now this should be `>-4.5` I think. Because 4.5 - 5.0.5 managers will not have this version installed.

Reply:
Yeah sure, you are right but this is was for testing and since we are not depending on 5.1 anymore for compat then this should be return back to 5.1.0